pkg/controller/podgc/gc_controller.go

```diff
@@ -168,7 +173,7 @@ func (gcc *PodGCController) gcTerminating(pods []*v1.Pod) {
 		wait.Add(1)
 		go func(namespace string, name string) {
 			defer wait.Done()
-			if err := gcc.deletePod(namespace, name); err != nil {
+			if err := gcc.deletePod(namespace, name, v1.DeletedFromUnusableNode); err != nil {
```

Comment:
It's worth adding a message in the condition: "Node tainted with node.kubernetes.io/out-of-service"